src/SFA.DAS.Roatp.CourseManagement.Web/Infrastructure/TempDataKeys.cs

```diff
@@ -4,5 +4,6 @@ public static class TempDataKeys
     {
         public const string SelectedAddressTempDataKey = "SelectedAddressTempDataKey";
         public const string DeleteProviderCourseDataKey = "DeleteProviderCourseDataKey";
+        public const string AddProviderCourseDataKey    = "AddProviderCourseDataKey";
```

Comment:
Stick to the convention, key names should be suffixed with TempDataKey, change the delete key name as well please